web/src/app/common/breakline.pipe.spec.ts

```diff
@@ -29,4 +29,12 @@ describe('BreaklinePipe', () => {
     const pipe = new BreaklinePipe();
     expect(pipe.transform('a\nb\nc')).toBe('a<br/>b<br/>c');
   });
+  it('return an empty string for undefined', () => {
```

Comment:
super nit: returns

I don't think it matters much so I won't leave comment for all occurrences. Feel free to ignore.

Reply:
Done